js/tests/enhancedDate.spec.js

```diff
@@ -0,0 +1,120 @@
+
+//Brian Forcum, Module_6 Homework
+
+describe("enhancedDate", function () {
+    
+    'use strict';
+
+    describe("setDate", function () {
+    
+        beforeEach(function () {
+                enhancedDate.setDate(new Date(2015, 4, 14));
+            });
+
+        it("Should be an instance of Date", function () {
+           expect(enhancedDate.setDate()).is.an.instanceof(Date);
+        });
+
+        it("Should not return null", function () {
+            expect(enhancedDate.setDate()).not.to.equal(null);
+        });
+
+    });
+
+    describe("getDate", function () {
+        var currentDate;
+
+        beforeEach(function () {
+                enhancedDate.setDate(new Date());
+                currentDate = new Date().getTime();
+               });
+
+        it("Should have a type of 'number'", function () {
+            expect(enhancedDate.getDate()).is.a("number");
+        });
+
+        it("Should be equal to current date", function () {
+            expect(enhancedDate.getDate()).to.equal(currentDate);
+        });
+    });
+
+    describe("getDayName", function () {
+        
+        beforeEach(function () {
+                enhancedDate.setDate(new Date(2015, 4, 14));
+               });
+
+        it("Should equal 'Thursday'", function () {
+
+           expect(enhancedDate.getDayName()).to.equal('Thursday'); 
+        });
+
+        it("Should return a string value", function() {
+            expect(enhancedDate.getDayName()).to.be.a("string");
+        });
+    });
+
+    describe("getMonthName", function () {
+    
+        beforeEach(function () {
+            enhancedDate.setDate(new Date(2014, 1, 19));
+        });
+
+        it("Should return 'January'", function () {
+            expect(enhancedDate.getMonthName()).to.equal("January");
+        });
+
+        it("Should not return a number", function () {
+            expect(enhancedDate.getMonthName()).is.not.a("number");
```

Comment:
This test is redundant with the one that follows it. If you want more tests, I'd simply test the return values for other known dates.